Translations/Portugues Brasil.csv

```diff
@@ -4,35 +4,35 @@ Key,Translation,Version,Comment (context for translators)
 <Mouse>,Mouse,,
 abandon,Descartar,,
 abort,Abortar,,
-aborting.expedition.died,Abortando expedição. O tripulante {0} morreu antes de entrar no transporte.,,
-aborting.expedition.target.too.far,Abortando expedição. Estamos muito longe do nosso alvo {0}.,,
+aborting.expedition.died,Abortando Expedição.{0} faleceu.,,

```

Comment:
De novo? Em inglês é `Aborting expedition. Crew member {0} died before entering the shuttle.`. Por que cortar a frase?

Reply:
explicaçoes demais , ai fica um texto enorme que poderia ser apenas resumido, muita coisas foi mudada pra ser mais informativo com menos palavras.